src/Sarif/Readers/SerializedPropertyInfoConverter.cs

```diff
@@ -34,7 +34,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             }
 
             string serializedValue = ((SerializedPropertyInfo)value).SerializedValue;
-            writer.WriteRawValue(@"""" + serializedValue + @"""");
+            writer.WriteRawValue(serializedValue);
```

Comment:
>serializedValue [](start = 33, length = 15)

it troubles me to see this change. i will talk to Larry about the history of this functionality. properties mgmt. has been a consistent source of parsing/validation errors. it would be very nice to put these behind us. i see you've added some new tests in this area, looking for Larry to validate that he's happy with the comprehensiveness of testing. may warrant a TODO, as this isn't the strict point of your change, obviously.

Reply:
Larry has logged #1673 for deeper testing here.